db/migrate/20180425123859_add_index_on_all_type_columns.rb

```diff
@@ -0,0 +1,61 @@
+class AddIndexOnAllTypeColumns < ActiveRecord::Migration[5.0]
+  def change
+    add_index :authentications, :type
+    add_index :availability_zones, :type
+    add_index :cloud_database_flavors, :type
+    add_index :cloud_databases, :type
+    add_index :cloud_networks, :type
+    add_index :cloud_resource_quotas, :type
+    add_index :cloud_subnets, :type
+    add_index :cloud_tenants, :type
+    add_index :cloud_volume_snapshots, :type
+    add_index :cloud_volumes, :type
+# no STI for configuration_locations ?
+#    add_index :configuration_locations, :type
+# no STI for configuration_organizations ?
+#    add_index :configuration_organizations, :type
+    add_index :configuration_profiles, :type
+    add_index :configuration_scripts, :type
+    add_index :configuration_tags, :type
+    add_index :configured_systems, :type
+    add_index :container_groups, :type
+    add_index :container_nodes, :type
+    add_index :container_volumes, :type
+    add_index :containers, :type
+    add_index :customization_scripts, :type
+    add_index :customization_templates, :type
+    add_index :dialog_fields, :type
+    add_index :ems_clusters, :type
+    add_index :ems_folders, :type
+    add_index :event_streams, :type
```

Comment:
Lets not add an index to `event_streams` unless we know it is needed for an index.
This table is BIG

Reply:
Removed `event_streams`